### PR TITLE
refactor: explain list models

### PR DESCRIPTION
**Pull request by bwanglzu**
A bit explanation on the support backbone model, attached 2 new models in the runner from @violenil 's transformer PR.

it is needed when writing documentation on the step-by-step guide.

4E947778-F22E-40AC-8F68-B0451650E0E7

### CODE REVIEW

**Comment by bwanglzu** [comment]
@jupyterjazz i'm not a big fan of loading extra resources files into the repo. the thing here is we need to always align the models here inside runner, this could be potentially an issue.

File: finetuner/__init__.py [hunk 1]
```diff
@@ -34,11 +34,55 @@ def login():
...
+            'task': 'text-to-text',
+            'output_dim': 768,
+            'details': 'Pretrained on BookCorpus and English Wikipedia.',
+        },
+        {
+            'model': 'sentence-transformers/msmarco-distilbert-base-v3',
+            'task': 'text-to-text',
+            'output_dim': 768,
+            'details': '''Pretrained on BookCorpus and English Wikipedia, has been
+            fine-tuned on the msmarco dataset from sentence-transformers.
```
Comment (guenthermi) [inline finetuner/__init__.py:83]:
```suggestion
            fine-tuned on the MS MARCO dataset from sentence-transformers.
```

**Review by guenthermi — APPROVED** [review]
LGTM, added only one minor comment. However, I think at some point, we should expose the available models from an endpoint of the API and it should be connected with the validation of the fine-tuning configuration.

File: finetuner/__init__.py [hunk 2]
```diff
@@ -34,11 +34,55 @@ def login():
...
+            'task': 'image-to-image',
+            'output_dim': 1280,
+            'details': 'Pretrained on ImageNet dataset.',
+        },
+        {
+            'model': 'efficientnet_b0',
+            'task': 'image-to-image',
+            'output_dim': 1280,
+            'details': 'Pretrained on ImageNet dataset.',
+        },
```
Comment (jupyterjazz) [inline finetuner/__init__.py:60]: Same models

**Pull request merged**